### ISSUE TITLE
List a couple of out-of-the-box supported fonts

### ISSUE THREAD

**Issue by igorsantos07** [powerline/fonts#195]
There are a couple of programming fonts that sport Powerline support symbols out of the box, such as:
- [Fira Code](https://github.com/tonsky/FiraCode)
- [Hasklig](https://github.com/i-tu/Hasklig)
- [Monoid](https://github.com/larsenwork/monoid)

Maybe those should be listed somewhere as "supported" fonts, or added to this repo?

**Comment by i-need-to-tell-you-something** [comment]
I agree. I came to the repo to CTRL+F to see if I can use Fira Code. Didn't get hits. Concluded it doesn't work with powerline. (And then decided to spend a few minutes on the issues list as well).

If someone skips (ore performs poorly) the last step they can come to the wrong conclusions.
